src/librustc/infer/error_reporting/nice_region_error/named_anon_conflict.rs

```diff
@@ -84,6 +86,13 @@ impl<'a, 'gcx, 'tcx> NiceRegionError<'a, 'gcx, 'tcx> {
             {
                 return None;
             }
+            if let FunctionRetTy::Return(ty) = &fndecl.output {
+                if let (TyKind::Def(_, _), ty::ReStatic) = (&ty.node, sub) {
+                    // This is an impl Trait return that evaluates de need of 'static.
```

Comment:
"evaluates _de_ need of 'static" ?

What is this?

Reply:
A partial rewording of the prior comment 😬

I believe something along the lines of "that evaluates to a need of static".